.github/workflows/release.yml

```diff
@@ -15,19 +15,21 @@ jobs:
     name: Publish to npm
     runs-on: ubuntu-latest
     environment: production
-
+    strategy:
+      matrix:
+        node-version: [20]
```

Comment:
The matrix strategy for such a workflow is redundant - you wouldn't (and couldn't) publish a package with multiple Node versions.

Reply:
Hi @ryami333 how would you suggest to setup the CI instead? Could you create a PR with your approach to discuss it?

We normally keep node versions to the latest LTS, do you see it beneficial to aim for multiple node versions? 

Thanks!

Reply:
The purpose of [matrix strategies](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#about-matrix-strategies) is "Running variations of jobs in a workflow". Configuring this job with a matrix strategy is redundant for two reasons:
- You've only put one element (20) in the matrix, so it's effectively the same as running it without a matrix strategy at all. You could have just written the Node version directly in the `actions/setup-node` step, as it was before this PR.
- You wouldn't _want_ to add any other versions to this particular workflow's matrix, because you wouldn't be able be publish the package two-or-more times. For testing workflows it makes sense to run the workflow with multiple Node versions (eg. `[18, 20, …etc]`) - once for each Node version that the library officially supports. But again - you can't publish a library multiple times.

Reply:
Hi @ryami333 that makes sense, thanks for the clear explanation.

I will create another PR approaching this topic, lets continue there